unstructured_ingest/v2/processes/connectors/lancedb/lancedb.py

```diff
@@ -134,6 +136,7 @@ async def run_async(self, path, file_data, **kwargs):
         async with self.get_table() as table:
             schema = await table.schema()
             df = self._fit_to_schema(df, schema)
+            await table.delete(f'{RECORD_ID_LABEL} = "{file_data.identifier}"')
```

Comment:
Can we check if it can be deleted by seeing it the column exists and log the warning the same way we did it here: [azure_ai_search](https://github.com/Unstructured-IO/unstructured-ingest/blob/main/unstructured_ingest/v2/processes/connectors/azure_ai_search.py#L281-L285)

Reply:
I logged in a different place but it makes more sense to do it here and not run the delete at all in that case.
So added the condition and moved the log.